.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
   ignorePatterns: [
     "**/_godot_defs/**",
-    "**/dummy/**",
+    "**/example/**",
```

Comment:
Hate to thrash you on the naming so much here, but could we rename `example` to `mockProject` or maybe `testProject` or something? (Having trouble thinking of a great name.) The reason I don't like `example` is because new people will probably expect it to be a sample ts2gd project, which it definitely isn't :)

Reply:
Sure, will change it to mockProject.